doc/source/rules.rst

```diff
@@ -0,0 +1,26 @@
+==========
+Game rules
+==========
+
+Each team owns two Bots, and each Bot is controlled by a Player. Bots in
+their homezone are called "Destroyers", and "Harvesters" when in the
+enemy’s homezone.
+
+**Eating**: when a Bot eats a food pellet, the food is permanently removed and
+*one point* is scored for that Bot’s team.
+
+**Eating another Bot**: when a Bot is eaten by an opposing destroyer, it
+returns to its starting position (as a harvester). *5 points*
+are awarded for eating an opponent.
+
+**Observations**: Bots can only observe an opponent’s exact position, if they
+or their teammate are within 5 squares of the opponent bot. If they are
+further away, the opponent’s positions are noised.
```

Comment:
We should say here that 5 squares does *not* mean that they are reachable in 5 moves, i.e. it is manhattan distance and not distance on the graph (students are often confused by this). And we should specify what kind of noise is added (I think it is a uniformly distributed integer between -5 and +5 on each coordinate? @Debilski ?)